### PR TITLE
New header should resize appropriately when there are page skins

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
When there is a page skin, the body and header should resize accordingly.

Please note page skins only happen at the wide breakpoint.

cc @ScottPainterGNM @guardian/commercial-dev 

## What is the value of this and can you measure success?
Looks less awful when there is a page skin.

## Does this affect other platforms - Amp, Apps, etc?
Nope

## Screenshots
**Before:**
image

**After:**
image

image

## Tested in CODE?
I will be testing in CODE and take screenshots
- [x] .

### CODE REVIEW

**Comment by NataliaLKB** [comment]
Okay screenshots have been added.

**Comment by gustavpursche** [comment]
Very nice.

I feel like the burger is now too close to "video" and there is too little space above the login area. Is that something Zef should look at?

**Comment by NataliaLKB** [comment]
> I feel like the burger is now too close to "video" and there is too little space above the login area. Is that something Zef should look at?

I agree, but I think maybe @zeftilldeath will have to look at it, only because it looks like the designs in figma have changed a bit and the desktop breakpoint on the live header has the same problems.  

image

File: static/src/stylesheets/layout/new-header/_feedback-form.scss [hunk 1]
```diff
@@ -19,6 +19,12 @@
 
     @include mq(wide) {
         width: $gs-column-width * 3;
+
+        .has-page-skin & {
+            width: auto;
+            position: relative;
+            left: 0;
```
Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_feedback-form.scss]: Just curious: What does the `left` property achieve?

Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_feedback-form.scss]: Only to even out the `right` setting https://github.com/guardian/frontend/pull/17329/files/dc758479884d87a22568e4a03b1a8ce12e086123#diff-35c6508f9586373237605727b5a950a3R17

😆

Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_feedback-form.scss]: tenor

Ah oh! Maybe add a `right: auto` for clearification?

**Review by gu-stav — APPROVED** [review]
Nice work!

**Comment by ScottPainterGNM** [comment]
Boom!

**Pull request merged**